PLAT-129603: Upgrade to React 17

hong6316 opened the pull request:
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

hong6316 committed:
Upgrade to React 17

Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

hong6316 committed:
Upgrade to React 17

Enact-DCO-1.0-Signed-off-by: Taeyoung Hong (taeyoung.hong@lge.com)

MikyungKim commented:
Please update PR title And rebase your branch with develop.

MikyungKim commented on packages/typescript/template/package.json:
```diff
@@ -36,14 +36,14 @@
     "@types/node": "^12.0.0",
-    "@types/react": "^16.8.0",
-    "@types/react-dom": "^16.0.11",
+    "@types/react": "^16.9.53",
```
I think the latest one is 17.0.0.

hong6316 committed:
Update typescript, remove tslint files

hong6316 committed:
Update typescript, remove tslint files

hong6316 commented:
to @MikyungKim ,
PR updated: Update typescript, remove tslint files

hong6316 committed:
Merge remote-tracking branch 'origin/develop' into feature/react17

hong6316 committed:
Update webosauto/template/package.json

hong6316 committed:
Update webosauto/template/package.json

hong6316 committed:
Update sandstone/template/package.json

sjro committed:
Update update-copyright-years.yml

sjro committed:
Don't need the update-copyright-years.yml file for develop branch.

MikyungKim committed:
Remove sandstone-related code for release (#28)

* Remove sandstone related code for release

Enact-DCO-1.0-Signed-off-by: Mikyung Kim (mikyung27.kim@lge.com)

* fix

Enact-DCO-1.0-Signed-off-by: Mikyung Kim (mikyung27.kim@lge.com)

hong6316 committed:
Delete package.json

MikyungKim committed:
3.2.0 release

MikyungKim committed:
3.2.0 release

MikyungKim committed:
3.2.0 release

MikyungKim committed:
3.2.1 release

MikyungKim committed:
3.2.1 release

MikyungKim committed:
3.2.1 release

hong6316 committed:
Merge branch 'feature/react17' of github.com:enactjs/templates into feature/react17
